frame/elections-phragmen/src/lib.rs

```diff
@@ -213,36 +213,42 @@ decl_storage! {
 	} add_extra_genesis {
 		config(members): Vec<(T::AccountId, BalanceOf<T>)>;
 		build(|config: &GenesisConfig<T>| {
-			let members = config.members.iter().map(|(ref member, ref stake)| {
-				// make sure they have enough stake
-				assert!(
-					T::Currency::free_balance(member) >= *stake,
-					"Genesis member does not have enough stake",
-				);
-
-				// reserve candidacy bond and set as members.
-				T::Currency::reserve(&member, T::CandidacyBond::get())
-					.expect("Genesis member does not have enough balance to be a candidate");
-
-				// Note: all members will only vote for themselves, hence they must be given exactly
-				// their own stake as total backing. Any sane election should behave as such.
-				// Nonetheless, stakes will be updated for term 1 onwards according to the election.
-				Members::<T>::mutate(|members| {
-					match members.binary_search_by(|(a, _b)| a.cmp(member)) {
-						Ok(_) => panic!("Duplicate member in elections phragmen genesis: {}", member),
-						Err(pos) => members.insert(pos, (member.clone(), *stake)),
-					}
-				});
-
-				// set self-votes to make persistent.
-				<Module<T>>::vote(
-					T::Origin::from(Some(member.clone()).into()),
-					vec![member.clone()],
-					*stake,
-				).expect("Genesis member could not vote.");
-
-				member.clone()
-			}).collect::<Vec<T::AccountId>>();
+			assert!(
+				config.members.len() as u32 <= T::DesiredMembers::get(),
+				"Cannot accept more than DesiredMembers genesis member",
+			);
```

Comment:
this is the only change isn't it ?

Reply:
yes, and I will revert the formatting. it is annoying that now it seems like I changed it all.